pandas/core/dtypes/inference.py

```diff
@@ -283,11 +285,13 @@ def is_list_like(obj):
     False
     """
 
-    return (isinstance(obj, compat.Iterable) and
+    return (isinstance(obj, compat.Iterable)
             # we do not count strings/unicode/bytes as list-like
-            not isinstance(obj, string_and_binary_types) and
+            and not isinstance(obj, string_and_binary_types)
```

Comment:
this is not correct, leave the and where it was

Reply:
PEP8 is clear about this (https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator)

Binary operators (like `and` is one) should come after a line-break. It's also more readable.

Reply:
Yes, changing this is in principle fine, we have been following that PEP8 rule recently (typically we only want such changes on lines that are already touched by the PR, but since you are here already touching the function some lines below, I would say it is fine).

Note that that is a recent change in PEP8, so you will see many places in the code that does it differently.